database/runtime/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM lambci/lambda:build-python3.8
+FROM public.ecr.aws/sam/build-python3.9:latest
```

Comment:
```suggestion
FROM --platform=linux/amd64 public.ecr.aws/sam/build-python3.9:latest
```

Reply:
i've had issues with deploying from the new mac, this solves it

Reply:
wait, not just mac, but also from github actions 🤔

Reply:
Even with `export DOCKER_DEFAULT_PLATFORM=linux/amd64`? This has been deploying from github actions as is. I'm reading up, though, to see if we should take out a branch from develop to make the change.

Reply:
Isn't github actions running on ubuntu so shouldn't need `--platform=linux/amd64`?

Reply:
thought so too, maybe i am misremembering. but it's always a good idea to add it whereever.